File: SymmetricNumberInput.c
#include <stdio.h>

int main() {
    int n;
    printf("Enter number of rows: ");
    scanf("%d", &n);

    for (int i = 1; i <= n; i++) {
        // Left side
        for (int j = 1; j <= i; j++) {
            printf("%d ", j);
        }

        // Spaces
        int spaces = 2 * (n - i);
        for (int s = 1; s <= spaces; s++) {
            printf("  ");
        }

        // Right side
        for (int j = i; j >= 1; j--) {
            printf("%d ", j);
        }

        printf("\n");
    }

    return 0;
}

File: SymmetricNumberInput.py
# Take input from user
n = int(input("Enter number of rows: "))

for i in range(1, n + 1):
    # Left side increasing numbers
    for j in range(1, i + 1):
        print(j, end=" ")

    # Spaces in the middle
    spaces = 2 * (n - i)
    print("  " * spaces, end="")

    # Right side decreasing numbers
    for j in range(i, 0, -1):
        print(j, end=" ")

    print()
